packages/@o3r/application/schemas/functional-content.metadat.schema.json

```diff
@@ -0,0 +1,306 @@
+              "type": "object",
+              "description": "Object definition for this section",
+              "required": [
+                "name",
+                "properties"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "Unique name of the definition in the section (which can be referenced by the rootReference property)"
+                },
+                "properties": {
+                  "type": "array",
+                  "description": "Array of properties of this definition",
+                  "minItems": 1,
+                  "items": {
+                    "type": "object",
+                    "required": [
+                      "name",
+                      "title",
+                      "description",
+                      "type"
+                    ],
+                    "additionalProperties": false,
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "description": "Unique id of the property"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "Title to display to the CMS user"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "Description to display to the CMS user"
+                      },
+                      "type": {
+                        "anyOf": [
+                          {
+                            "$ref": "#/$defs/enumProperty"
+                          },
+                          {
+                            "$ref": "#/$defs/objectProperty"
+                          },
+                          {
+                            "$ref": "#/$defs/primitiveProperty"
+                          }
+                        ]
+                      },
+                      "widget": {
+                        "$ref": "#/$defs/widget"
+                      }
+                    }
+                  }
+                },
+                "idPropertyReference": {
+                  "type": "string",
+                  "description": "When the definition is used in an array/map defines the name of the property that should be used to uniquely identify the definition in the array/map"
+                },
+                "titlePropertyReference": {
+                  "type": "string",
+                  "description": "When the definition is used in an array/map defines the name of the property that should be used to display a user friendly label for the definition in the array/map"
+                },
+                "widget": {
+                  "$ref": "#/$defs/widget"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "$defs": {
```

Comment:
I don't think it is part of  `http://json-schema.org/schema`, usually we use `definitions` instead